### ISSUE TITLE
Broken as of Home assistant 0.115 update

### ISSUE THREAD

**Issue by helenclarko** [keitetran/BroadlinkIRTools#21]
As of HA 0.115 update the broadlink integration no longer uses broadlink.learn.

Instead remote.learn_command is used.

**Comment by l31ghj** [comment]
@keitetran would you be able to change this to use the new command?

**Comment by Paulie-Moo** [comment]
Sadly not working for me either

**Comment by huykaamaa** [comment]
core-2021.2.0 ver cũng vậy ạ

**Comment by ninadpchaudhari** [comment]
Nope, The new "learn_command" stores the codes inside the storage folder, It does not return the code back... hence its basically impossible to use this I think.

**Comment by LostOnTheLine** [comment]
I know this is old but...
Why not have a way for us to paste in our codes from our `broadlink_remote_MACADDRESS_codes` file?
I understand things changed & it can't get the commands sent back, but this tool would still be usable if it was able to just paste in the codes instead of getting them from the device
